% Template for PLoS
%M13 genes
%I(assembly),
%II(replication init),
%X (DNA accumulation, inside gII) 
%III(fusion),
%IV(assembly),
%V(destabilizing protein),
%VI(minor coat),
%VII(minor coat),
%VIII(major coat),
%IX(minor coat),
%subsubsection is 3rd level heading, PLOS doen't support 4th level.
% Version 3.3 June 2016
% Please follow the template instructions whenever possible.
%
% Once your paper is accepted REMOVE ALL TRACKED CHANGES.
% Use latexdiff to track changes during review to maintain a clean file.
% Visit https://www.ctan.org/pkg/latexdiff?lang=en for info
%
% There are no restrictions on package use within the LaTeX files except that
% no packages listed in the template may be deleted.
% Do not include colors or graphics in the text.
%
% LaTeX source should be a single file: don't use \input, \externaldocument
%
% -- FIGURES AND TABLES
%
% Include table/figure captions directly after paragraph where first cited.
%
% NO GRAPHICS
% - Figures should be uploaded separately from your manuscript file. 
% - Extract figures (in PDF) generated by LaTeX before submission.
% - Combine multiple panels/subfigures into one image before submission.
% For figure citations, please use "Fig" instead of "Figure".
% See http://journals.plos.org/plosone/s/figures for PLOS figure guidelines.
%
% Tables should be cell-based and may not contain:
% - spacing/line breaks within cells to alter layout or alignment
% - do not nest tabular environments (no tabular environments within tabular environments)
% - no graphics or colored text (cell background color/shading OK)
% See http://journals.plos.org/plosone/s/tables for table guidelines.
%
% For tables that exceed the width of the text column, use the adjustwidth environment as illustrated in the example table in text below.
%
% % % % % % % % % % % % % % % % % % % % % % % %
%
% -- EQUATIONS, MATH SYMBOLS, SUBSCRIPTS, AND SUPERSCRIPTS
%
% IMPORTANT
% Below are a few tips to help format your equations and other special characters according to our specifications. For more tips to help reduce the possibility of formatting errors during conversion, please see our LaTeX guidelines at http://journals.plos.org/plosone/s/latex
%
% For inline equations, please be sure to include all portions of an equation in the math environment.  For example, x$^2$ is incorrect; this should be formatted as $x^2$ (or $\mathrm{x}^2$ if the romanized font is desired).
%
% Do not include text that is not math in the math environment. For example, CO2 should be written as CO\textsubscript{2} instead of CO$_2$.
%
% Please add line breaks to long display equations when possible in order to fit size of the column. 
%
% For inline equations, please do not include punctuation (commas, etc) within the math environment unless this is part of the equation.
%
% When adding superscript or subscripts outside of brackets/braces, please group using {}.  For example, change "[U(D,E,\gamma)]^2" to "{[U(D,E,\gamma)]}^2". 
%
% Do not use \cal for caligraphic font.  Instead, use \mathcal{}
%
% % % % % % % % % % % % % % % % % % % % % % % % 
%
% Please contact latex@plos.org with any questions.
%
% % % % % % % % % % % % % % % % % % % % % % % %

\documentclass[10pt,letterpaper]{article}
\usepackage[top=0.85in,left=2.75in,footskip=0.75in]{geometry}

% amsmath and amssymb packages, useful for mathematical formulas and symbols
\usepackage{amsmath,amssymb}

% Use adjustwidth environment to exceed column width (see example table in text)
\usepackage{changepage}

% Use Unicode characters when possible
\usepackage[utf8]{inputenc}
%\usepackage[utf8x]{inputenc}

% textcomp package and marvosym package for additional characters
\usepackage{textcomp,marvosym}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}

% Use nameref to cite supporting information files (see Supporting Information section for more info)
\usepackage{nameref,hyperref}

% line numbers
\usepackage[right]{lineno}

% ligatures disabled
\usepackage{microtype}
\DisableLigatures[f]{encoding = *, family = * }

% color can be used to apply background shading to table cells only
\usepackage[table]{xcolor}

% array package and thick rules for tables
\usepackage{array}

% create "+" rule type for thick vertical lines
\newcolumntype{+}{!{\vrule width 2pt}}

% create \thickcline for thick horizontal lines of variable length
\newlength\savedwidth
\newcommand\thickcline[1]{%
  \noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
  \cline{#1}%
  \noalign{\vskip\arrayrulewidth}%
  \noalign{\global\arrayrulewidth\savedwidth}%
}

% \thickhline command for thick horizontal lines that span the table
\newcommand\thickhline{\noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
\hline
\noalign{\global\arrayrulewidth\savedwidth}}


% Remove comment for double spacing
%\usepackage{setspace} 
%\doublespacing

% Text layout
\raggedright
\setlength{\parindent}{0.5cm}
\textwidth 5.25in 
\textheight 8.75in

% Bold the 'Figure #' in the caption and separate it from the title/caption with a period
% Captions will be left justified
\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}
\renewcommand{\figurename}{Fig}

% Use the PLoS provided BiBTeX style
\bibliographystyle{plos2015}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother

% Leave date blank
\date{}
% Header and Footer with logo
\usepackage{lastpage,fancyhdr,graphicx}
\usepackage{epstopdf}
\pagestyle{myheadings}
\pagestyle{fancy}
\fancyhf{}
\setlength{\headheight}{27.023pt}
%\lhead{\includegraphics[width=2.0in]{PLOS-submission.eps}}
%\rfoot{\thepage/\pageref{lastpage}}
\renewcommand{\footrule}{\hrule height 2pt \vspace{2mm}}
\fancyheadoffset[L]{2.25in}
\fancyfootoffset[L]{2.25in}
%\lfoot{\sf PLOS}
\lfoot{\sf NCMLS}

%% Include all macros below

\newcommand{\lorem}{{\bf LOREM}}
\newcommand{\ipsum}{{\bf IPSUM}}

%% END MACROS SECTION


\begin{document}
\vspace*{0.2in}

% Title must be 250 characters or less.
\begin{flushleft}
{\Large
\textbf\newline{Mutation Rate as a Selection Mechanism in Continuous Evolution of Bacteriophage} % Title Case
}
\newline
% Insert author names, affiliations and corresponding author email (do not include titles, positions, or degrees).
\\
Peter Reintjes%\textsuperscript{1,2}
\\
\bigskip
\textbf{Innatrix, Inc.}
\bigskip

% Additional author notes using the symbols described below. Insert symbol callouts after author names as necessary.
% 
% Remove or comment out the author notes below if they aren't used.
%
% Primary Equal Contribution Note
%\Yinyang These authors contributed equally to this work.

% Additional Equal Contribution Note
% Also use this double-dagger symbol for special authorship notes, such as senior authorship.
%\ddag These authors also contributed equally to this work.

% Current address notes
%\textcurrency Current Address: Dept/Program/Center, Institution Name, City, State, Country % change symbol to "\textcurrency a" if more than one current address note
% \textcurrency b Insert second current address 
% \textcurrency c Insert third current address

% Deceased author note
%\dag Deceased

% Group/Consortium Author Note
%\textpilcrow Membership list can be found in the Acknowledgments section.

% Use the asterisk to denote corresponding authorship and provide email address in note below.
%* correspondingauthor@institute.edu

\end{flushleft}
% Please keep the abstract below 300 words
\section*{Abstract}
Engineering novel proteins via continuous evolution of bacteriophage currently requires the host bacteria to be transformed with two additional functions: A mutagenesis vector to provide an elevated rate of viral mutation, and a selection mechanism to increase the number of infectious progeny for the individual encoding the improved product\cite{pace}.  I propose using a mutagenesis suppressor as the selection mechanism whereby the desired activity results in a higher number of clones, rewarding the genotype by increasing the number of identical progeny.

% Please keep the Author Summary between 150 and 200 words
% Use first person. PLOS ONE authors please skip this step. 
% Author Summary not valid for PLOS ONE submissions.   
%\section*{Author Summary}
%Lorem ipsum dolor sit amet,

%\linenumbers

% Use "Eq" instead of "Equation" for equation citations.
\section*{Introduction}
Continuous evolution of bacteriophage has the potential to become a potent protein engineering tool\cite{pace}\cite{monsanto}.
Continuous evolution rapidly produces a viral genome containing a gene
which has undergone many generations of mutation and selection for
a particular property.  The generality of this approach to protein engineering
is limited by our ability to insert an expressable initial gene into the phage and create a selection mechanism for the desired activity.

Specifically, the Phage Assisted Continuous Evolution (PACE)\cite{pace} system requires:
\begin{itemize}

\item{}
A modified viral genome replacing a crucial phage gene with the gene to be evolved.

\item{}
A transformed host with inducible mutagenesis.

\item{}
A host plasmid containing a selection mechanism to provide the crucial phage gene in proportion to the desired activity of the evolving gene.
\end{itemize}

\section{M-Selection}
Extremely high, yet controllable, in vivo mutation rates are now possible\cite{mutation}.  This broad-spectrum mutagenesis with a (claimed) 320,000:1 dynamic range is generated and can be controlled within an individual host cell.  I suggest that if this mutagenesis is initiated  by infection, say with the phage shock promotor but then reduced by the desired activity, the two principal components of evolution will be satisfied: The desired activity will be selected for as the number of phage progeny of the individual exhibiting that activity will be amplified by more faithful reproduction.  I propose calling this mechanism Mutagenesis-selection or M-selection.  So far, all PACE-derived procedures induce a uniform mutagenesis in the lagoon: M-selection doesn't even require the small-molecule (Ara) induction mechanism.

We require 2 clones per generation to infect hosts and produce phage to prevent washout of a genotype. Assuming phage production of 100/hour and a lagoon transit time of one hour, we calculate calculate a mutation rate of about 4 mutations per phage genome.  The percentage of phage progeny which are exact copies of the parent is given by the Poisson distribution where $\mu = 0$ is the expected number of mutations and $\lambda$ is the mutation rate per virion (mutation rate/base * 6kbp/ genome).
A mutation rate of $\lambda = 4$ gives a 1.8\% probability of zero mutations.
\vskip 0.2cm
\begin{eqnarray}
\label{eq:POC}
\mathrm{P}\left( \mu, \lambda \right) = \frac{{e^{ - \lambda } \lambda ^ \mu }}{{\mu!}}
 \rightarrow P(0,4) = \frac{{e^{ - 4 }}}{{1}} = .018
\end{eqnarray}

So the maximum per-base mutation rate that a genome could tolerate and still stay in the lagoon is $6400/4 = 6.25 X 10^-4$.
Liu et al. claim 322,000 times basal mutation rate. Basal rate is possibly ambiguous here if they are referring to the basal rate of E. coli producing E. coli  ($4x10-10$) vs. E coli producing M13 phage ($7.2 x 10-7$), but in either case their claimed dynamic range covers the required phage mutation crossover rate required to avoid washout.

PACE is designed to keep the transit time of host cells through the lagoon on the order of one generation.  However, at very high levels of mutation it seems unlikely that succeeding generations of E. coli mutants or phage will remain viable. Furthermore, the continuous flow environment ensures that there can be no other selection mechanism beyond the number of faithful copies of a particular genome.  The obvious exception to this would be a colony or biofilm which remains somehow attached in the lagoon, but the population size for such entities would be limited by the surface areas within the lagoon, and not the volume.

Taking a phage production average of 100/hour for normal M13 infection requires us to have a cloning rate of at least 2\%: two faithful copies per infection in order to avoid washout.  The previous consideration of high mutation rates would seem to allow slower flow rates, which will lower this minimum fraction.

An important difference with PACE is that mutagenesis is not induced externally and therefore in uninfected cells, so there may be a minimum number of faithful copies produced before the mutagenesis products are expressed by the infected host.  It may prove useful to retain the arabinose-induced mutagenesis as a way of achieving a baseline mutation rate in the lagoon, or an ante-chamber for the lagoon where induction can begin before exposure to phage.

As the evolving protein approaches a high level of desired activity, the proportionally lowered mutation rate will ensure that more viral progeny contain exact copies of the parent genotype, while a baseline mutation rate ensures that our mechanism to lower the mutation rate will not result in stagnation.

A possible drawback to this technique is that the mutation space is less agressively searched as the desired activity improves.  But there is no particular reason that the modulation of mutation rate necessary to produce the population shift will be a particularly low (e.g. natural) mutation rate.

\section{Possible Benefits of M-Selection}
\begin{enumerate}
\item{The virus is not crippled, that is, no critical genes are removed from the virion which should improve our ability to propagate phage prior to the evolution experiment.  Care must be taken to ensure that the evolving gene is not discarded, but M13 plus one additional gene or domain is exactly what has been perfected in phage display.}
\item{No phage genes are required in a host plasmid.  This avoids the problem associated with the presence of gIII in the host.}
\item{High rates of mutation will be associated with individuals exhibiting little or none of the desired activity, and these lines of descent (both E. coli and phage) will be subject to catastrophic levels of mutation and low viability. This may reduce the importance of a carefully controlled flow rate.}
\item{Clonal Interference\cite{dynamics}, the inabiltiy of a single genotype to gain sufficient advantage to become fixed in a population due to weak selection pressure is viewed as a problem in experimental evolution (one mutant has an important beneficial mutation, but also has a bad mutation hitch-hiking along which attenuates the advantage as compared to a sibling with neither mutation. In our case, maybe not a disadvantage as it keeps variety in the lagoon. Multiple genotypes in the lagoon at sample time would be informative. Does weak selection correspond to a flatter fitness landscape? I'm making an assumption here that M-selection corresponds to what Drake and others refer to as 'weak selection'.}

\item{We get 'genetic drift' for free, or at the cost of defective interference, anyway. To quote Patent \#9023549 (Havard/PACE):
\quote{\textit{A major problem with traditional directed evolution, whether in vitro or in vivo, is that libraries that do not contain any functional variants will be entirely lost, and the effort wasted--even if functionality lies only a couple of mutations away. The present invention overcomes this problem by allowing "switchable" genetic drift. This can be achieved by, for example, providing "free" propagation components to all library members, such as by inducible expression from an anhydrotetracycline-regulated titratable promoter, enabling all variants to infect host cells. By providing enough "free" propagation components, but less than the optimal level, any functional variants that arise will produce slightly more of the missing propagation components, produce more infectious progeny, and take over the population. }}}

\item{Adding a negative selection component may be simpler than in PACE: Just need something to re-up-modulate the mutation rate as a result of off-target binding. A suppressor supressor?}

\end{enumerate}

The nominal (high) mutation rate can be chosen to guarantee that there will be a minimum number of non-mutant copies produced for every infection.  This percentage (of clones) will then increase proportunally as the mutation rate is reduced.

The downside to this is defective interference from all the propagating mutants with none of the desired activity. So they are infecting host cells and crowding out the 'nice clones' as we wait for their lineages to undergo Error Catastrophe.  It would be good to know how many uninfected host are making it through the lagoon. If we have an excess of uninfected host, this would reduce defective interference.

In the first generation, even a fatal mutation will be packaged in a viable virion, so that it will infect a host cell. But it will go no further if the fatal mutation results in non-functional coat proteins, origin of replication, packaging signal, or whatever. So in the worst case, a fatal mutation results in the production of 100 dead-end virions.

The maximum mutation rate will be experienced by phage that have none of the desired activity and we want this rate to ensure that the accumulation of 5 non-beneficial mutations will result in a fatality.  The growth of a defectively interfering population is thus limited to 100^5, 10^6.  This number is reached only if for phage that has accumulated four non-beneficial, non-fatal mutations.  Presumably these are silent mutations as the phage genome is minimal and highly conserved. We should also recognize that a reversion to wild-type phage (shedding the payload gene), will also subject the phage to maxiumum mutation in every generation.

Beneficial mutations resulting in detection of the desired activity will reduce the mutation rate, resulting in more cloned phage, which will inherent this ability to supress the mutation rate.

Even if it is possible to have a series of mutations which result in a viable phage after ten generations, this genotype will not.  For the high-mutation
all of the different mutations that occur in these 10 generations would have to be non fatal. resulting in 10^10 viable phage mutations.

For a given maximal mutation rate, what are the
Can we estimate a probability for M13s resistance to mutation.
Since different types of organisms, and even individual species, have some control over their exposure to mutations.

Perfect replication okay for Artificial Selection

There is at least one aspect of natural evolution which we do not have to replicate in directed evolution.  The principals of evolution make it clear that perfect DNA replication is a recipe for extinction in any environment that is subject to change.  But we are clearly going to engineer our continuous evolution environment to be as stable as possible, so future experiments can take advantage of DNA replication and repair mechanisms to allow for perfect replication at the upper end of the usable range.

Phage has a higher natural mutation rate than all types of cells, but our artificial environment does not have to respect
There is no reason not to utilize the entire range from Mutation Storm to Perfect Replication.  In fact, modifications which can achieve perfect replication.

So we may do something nature never did, which is bring all the tools of replicative fidelity to phage reproduction.

What fraction of the M13 gene can carry non-fatal mutations. Including all the silent mutations, plus those which represent similar amino acid substitutions,


\begin{minipage}{\textwidth}
Figure 1 shows the combined mutagenic/selection mechanism replacing the Mutagenesis Plasmid(MP) and Accessory Plasmid (AP) in PACE.

\def\boxone#1{
  \put(19,30){\line(1,0){67}}     % horiz top
  \put(19,0){\line(1,0){67}}      % horiz bottom
  \put(20,0){\line(0,1){30}}      % vert left
  \put(85,0){\line(0,1){30}}      % vert right
  \put(25,15){#1}
}
\def\boxtwo#1#2{
  \put(19,30){\line(1,0){80}}     % horiz top
  \put(19,0){\line(1,0){80}}      % horiz bottom
  \put(20,0){\line(0,1){30}}      % vert left
  \put(98,0){\line(0,1){30}}      % vert right
  \put(25,17){#1}
  \put(25,7){#2}
}
\def\circleone#1#2{\circle{#1}\put(-17,-4){#2}}
\def\circletwo#1#2#3{\circle{#1}\put(-17,0){#2}\put(-15,20){#3}}

\picture(400,150)(-90,0)

\put(90,90){\vector(0,-1){12}}
\put(90,56){\vector(0,-1){26}}

\linethickness{2pt}
\thicklines
\put(0,15){\circleone{40}{\Large{\ \ P1}}}
\put(0,35){\vector(0,1){16}}
\put(10,40){\small Protein}
\put(20,15){\vector(1,0){34}}
\put(25,18){DNA}

\put(35,90){\boxtwo{\ Phage Shock}{\ \ \ Promoter}}
\put(-20,62){
\fbox{%
    \begin{minipage}{1.0in}
Activity Induced\linebreak
\hspace*{5 mm}Repressor
    \end{minipage}%
}}
\put(64,70){\vector(1,0){18}}
\put(83,65){\fbox{\begin{minipage}{0.05in}
		\hfill\vspace{0.20in}
            \end{minipage}
           }}
\put(35,0){\boxtwo{\ \ Mutagenic}{\ \ Components}}

\put(134,15){\vector(1,0){26}}
\put(180,15){\circleone{40}{\Large{\ \ F1}}}

\endpicture
\vskip 1.0cm
\centerline{\bf Fig 1. M-selection Plasmid}
\end{minipage}
\vskip 0.2cm

\section{Comparison with Natural Selection}
Selection pressure for an optimal mutation rate has its own natural history\cite{evomut} although for settled phyla we see consistent\cite{drake91}, low mutation rates.  Bacteriophage $6.4 X 10^-4$, and multicellular eukaryote mutation rates are higher than bacteria $4 X 10^-11$ and single cell eukaryotes $3 X 10^-10$.

A selection mechanism for lowered mutation rate would not seem to correspond to anything in the natural world.  Natural selection is largely about responses to the external environment, which we control closely in laboratory evolution.  Perhaps the closest parallel is found in species which can reproduce by asexual or sexual reproduction and change the rates between the alternatives as a way of accessing more diversity through sexual reproduction or producing more clones via asexual reproduction.


\section*{Comparison with Sexual/Asexual Reproduction}
Sexual reproduction is not the same as an elevated mutation rate, but it shares the characteristic that even a perfectly fit individual will not be producing faithful copies of its genome. For organisms with access to both forms, the percentage of offspring produced by asexual reproduction would be comparable to our percentage of true copies.
\subsection*{Yeast}
Although yeast has access to both sexual and asexual reproduction, the time and energy cost of sexual reproduction is high and it is generally induced by stress, indicating a need for more genetic diversity.
\subsection*{Daphnia} 
In Daphnia, longevity and healthy survival to the end of the season result in so-called ``winter eggs'' which reproduce asexually, providing a percentage of offspring which are clones of the parent. The cost differential between sexual and asexual reproduction is low and each generation represents a mix of both.

\section*{Materials and Methods}

\subsection*{Population Dyamics of Bacteriophage (Under Construction)}
\subsubsection*{Clones versus Mutants}
Although it may sounds like the title of a bad sci-fi movie, this precisely states the selection mechanism of mutation.
\newline
Percentage of activated genes due to Phage Shock promoter.
\newline
Number of polIII-(mutagenic polymerase) transcripts times error rate due to mutagenic polymerase.
\newline
Error rate of normal polII, polIII?
\newline
Number of phage particles produced np(0-6 min) = 0 (six minutes before any phage appears), then production between 75-200 per hour.
\newline
Some initial single-strand (virion) DNA production will occur in non-mutagenic environment, then mutants appear, until suppression of mutagenesis due to binding (e.g. the desired activity).

% Results and Discussion can be combined.
\section*{Results}
\includegraphics{{0.0cAMP_M13_MP6-7View}.jpg}
\newline
M13 with 0.0 cAMP + MP6 $10^-7$
\includegraphics{{5mMcAMP_M13_MP6-5View}.jpg}
\newline
M13 with 5 milliMolar cAMP + MP6 $10^-5$
\includegraphics{{50mMcAMP_M13_MP6-5View}.jpg}
\newline
M13 with 50 milliMolar cAMP + MP6 $10^-5$


\subsection{Methods for Lowering the Mutation Rate}
\subsubsection*{RNA Anti-sense}

The components for the broad-spectrum, high-level mutagenisis described in Badran and Liu\cite{mutation} are variations (often single mutation variants) of the functional domains of high-fidelity DNA replication, and as such are not suitable targets for RNA anti-sense interference. Any such anti-sense interference would be likely to affect their normal counterparts which are needed for faithful reproduction.

RNA anti-sense of gIV could be utilized to block the phage shock promotor which is the primary stimulus that activates mutagenesis.

\subsubsection*{Other Methods}
\begin{itemize}
	\item Suppress pIV production or otherwise block phage shock promotor.
	\item Selectively block downstream components of mutagenesis.
	\item Enhance production of wild-type high-fidelity replication components.
\end{itemize}

%\section*{Discussion}
%Modeling needs work.
%\section*{Conclusion}
%
%I think this will work.
%\section*{Supporting Information}

% Include only the SI item label in the paragraph heading. Use the \nameref{label} command to cite SI items in the text.
%\paragraph*{S1 Fig.}
%\label{S1_Fig}
%{\bf Bold the title sentence.} Add descriptive text after the title of the item (optional).

%\paragraph*{S1 Appendix.}
%\label{S1_Appendix}
%{\bf Algorithm.} The code goes here.

%\paragraph*{S1 Table.}
%\label{S1_Table}
%{\bf Graph maybe.} The Graph goes here.

%\section*{Acknowledgments}
%So many people to thank.

\nolinenumbers

% Alternative to directly entering bibtex entries:
% 1) Compile BiBTeX database using plos2015.bst style file
% 2) Paste the contents of your .bbl file here.
\begin{thebibliography}{10}

\bibitem{pace}
Esvelt K. M., Carlson J. C. \& Liu D. R.
\newblock \textit{A system for the continuous directed evolution of biomolecules}.
\newblock Nature 472, 499–503 (2011).

\bibitem{mutation}
Badran AH, Liu DR.
\newblock \textit{{D}evelopment of potent in vivo mutagenesis plasmids with broad mutational spectra}.
\newblock Nature Communications. 2015;6:8425. doi:10.1038/ncomms9425.

\bibitem{dynamics}
Barrick, Jeffrey E., Lenski, Richard E.
\newblock \textit{{G}enome Dynamics during Experimental Evolution}
\newblock  Nat Rev Genet, 2013 December, 14(12), pp. 827-839, doi:10.1038/nrg3564

\bibitem{evomut}
Lynch M.
\newblock \textit{{E}volution of the Mutation Rate}.
\newblock Trends in Genetics, 2010 August; 26(8): 345–352. doi:10.1016/j.tig.2010.05.003.

\bibitem{monsanto}
  Badran A. H., Guzov V.M., Huai Q., Kemp M. M.,Vishwanath P.,Kain W., Evdokimov A., Moshiri F., Turner K.H.P., Malvar T., Liu D.R.
  \newblock \textit{{C}ontinuous evolution of Bacillus thuringiensis toxins overcomes insect resistance.}
  \newblock  Nature 533,58-63 (2016) doi:10.1038/nature17938.

\bibitem{drake91}
  Drake J. W., 
  \newblock \textit{{A} constant rate of spontaneous mutation in DNA-based microbes}, 
  \newblock  Proc Natl Acad Sci U S A. 1991 Aug 15;88(16):7160-4.

\bibitem{domingo2009}
  Domingo-Calap P, Cuevas JM, Sanjua´n R (2009)
\newblock The Fitness Effects of Random Mutations in Single-Stranded DNA and RNA Bacteriophages.
\newblock PLoS Genet 5(11): e1000742. doi:10.1371/journal.pgen.1000742

\end{thebibliography}

\fbox{\begin{minipage}{4.0in}
Bacteriophage M13. The genome size is 6407 bp (4). The
mutational target is 258 bp of an inserted Escherichia coli
lacZa sequence (5). The spontaneous mutant frequency is $6.4 x 10-4$ per genome or $7.2 x 10-7$ per base pair \textit{Drake, 1991}
\end{minipage}}

\end{document}



